VsViewer/vs_calc/SPT.py

```diff
@@ -101,7 +101,7 @@ def from_file(spt_ffp: str):
         """
         spt_ffp = Path(spt_ffp)
         data = pd.read_csv(spt_ffp)
-        soil_type = None if "Soil" in data.columns else data["Soil"]
+        soil_type = None if "Soil" not in data.columns else data["Soil"].map(lambda x: SoilType[x])
```

Comment:
Quick question, what is this line doing in lay person terms?

Reply:
What this is doing is first checking if "Soil" is in the files columns. If it is not in there then it sets the soil_type to None so would use defaults. Otherwise it will go through the entire column and map each entry to the SoilType Enum to ensure there is no typo issues for classification for soil types.